src/hooks/shards/useShardHydration.ts

```diff
@@ -12,10 +14,16 @@ import {
     useShardDetail_setShards,
 } from 'src/stores/ShardDetail/hooks';
 
+interface UseShardHydrationSettings {
+    defaultMessageId?: ShardStatusMessageIds;
+}
```

Comment:
Initially I needed to add a setting but no longer need to but leaving this in for future proof.